python/example_code/keyspaces/README.md

```diff
@@ -0,0 +1,111 @@
+# Amazon Keyspaces code examples for the SDK for Python
+
+## Overview
+
+Shows how to use the AWS SDK for Python (Boto3) to work with Amazon Keyspaces 
+(for Apache Cassandra).
+
+*Amazon Keyspaces is a scalable, highly available, and managed Apache Cassandra–compatible 
+database service.*
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Hello
```

Comment:
I'm not opposed to "Hello" as the header, but checking to see if "Get started" was intended here (per the [template](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template)).

Reply:
Yes! fixed.